pkg/repository/profileManager.go

```diff
@@ -0,0 +1,119 @@
+package repository
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+	awsRepo "github.com/CollActionteam/collaction_backend/pkg/repository/aws"
+	"github.com/CollActionteam/collaction_backend/utils"
+)
+
+type Profile struct {
+	dbClient awsRepo.DynamoDb
```

Comment:
This code has dependencies on AWS library types, so it should be in the "aws" sub-directory